### PR TITLE
Simplify `F.selu` test

### PR DESCRIPTION
**Pull request by aksub99**
Simplified F.selu test using testing.FunctionTestCase.
Ref. #6071

### CODE REVIEW

File: tests/chainer_tests/functions_tests/activation_tests/test_selu.py [hunk 1]
```diff
@@ -1,71 +1,57 @@
...
-
-    @attr.gpu
-    @condition.retry(10)
-    def test_backward_gpu(self):
-        self.check_backward(cuda.to_gpu(self.x), cuda.to_gpu(self.gy))
+    def forward_expected(self, inputs):
+        x, = inputs
+        expected = numpy.where(x >= 0, x, self.alpha * (numpy.exp(x) - 1))
+        expected *= self.scale
+        return expected.astype(x.dype),
```
Comment (toslunar) [inline tests/chainer_tests/functions_tests/activation_tests/test_selu.py]: Could you fix typo?
```suggestion
        return expected.astype(x.dtype),
```

Comment (aksub99) [reply tests/chainer_tests/functions_tests/activation_tests/test_selu.py]: Fixed!

**Comment by toslunar** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
The Jenkins failure is unrelated to the PR.

**Pull request merged**